### PR TITLE
fixed macOS requirements.

### PR DESCRIPTION
**Pull request by paxcoder**
Hey.
did you noticed that macOS build was broken?
this should fix it.

### CODE REVIEW

File: requirements-mac.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 wxpython
-pywin32
```
Comment (yncat) [inline requirements-mac.txt:2]: Who the hell added pywin32 dependency to mac... It was probably me. Obviously copied from win and forgot to delete. Sorry and thanks.

**Review by yncat — APPROVED** [review]
LGTM!

**Comment by paxcoder** [comment]
Ok, but there’s issue with newest pyinstaller, could you fix it?
I wanted to compile under m1.

**Comment by yncat** [comment]
Sorry, I don't have M1 macs...

**Comment by paxcoder** [comment]
Are there arm versions of the libs it uses?
If yes, could you just add them in, for m1?
I can handle testing.

**Pull request merged**